docker/conf/traefik.toml

```diff
@@ -37,10 +37,10 @@ insecureSkipVerify = true
     address = ":80"
   [entryPoints.https]
     address = ":443"
-    [entryPoints.https.tls]
-      [[entryPoints.https.tls.certificates]]
-      certFile = "/etc/traefik/sni/cert/altis.pem"
-      keyFile = "/etc/traefik/sni/key/altis.pem"
+      [entryPoints.https.tls]
```

Comment:
Is this indentation change important?

Reply:
Not syntactically, no. Was following indentation behavior in previous lines, and probably copy/pasted like that as well.

Reply:
Figured out what you meant 🤦 and fixed it.